src/models/actor.ts

```diff
@@ -1,24 +1,142 @@
 import type { ISystemFields } from './system-fields';
 
 export interface IActor {
+  adres: IActorAdres;
+  adressen: IActorAdres[];
+  afkorting: string;
+  emails: IEmail[];
+  erkenningen: IErkenning[];
   id: number;
-  uri: string;
+  ids: IId[];
+  info: unknown[];
+  naam: string;
+  omschrijving: string;
+  opmerkingen: string;
+  relaties: IRelatie[];
   self: string;
-  type: {
-    id: number;
-    naam: string;
-    uri: string;
-  };
-  zichtbaarheid: {
-    id: number;
-    naam: string;
-  };
+  status: IActorStatus;
+  systemfields: ISystemFields;
+  telefoons: ITelefoon[];
+  type: IType;
+  types: string[];
+  uri: string;
+  urls: string[];
+  voornaam: string;
+  zichtbaarheid: IType;
+}
+
+interface IActorAdres {
+  gemeente: IGemeente;
+  land: ILand;
+  postcode: IPostcode;
+  straat: IStraat;
+  adres: IAdresregisterAdres;
+}
+
+interface ILand {
+  code: string;
+  naam: string;
+}
+
+interface IGemeente {
+  id?: number;
+  naam: string;
+  niscode: string;
+  provincie?: IProvincie;
+}
+
+interface IPostcode {
+  nummer: string;
+  uri: string;
+}
+
+interface IStraat {
+  id: string;
+  naam: string;
+  uri: string;
   omschrijving: string;
+}
+
```

Comment:
ik twijfel hier wel of al die adres dingen niet beter gemeenschappelijk zitten, ofwel degene hier prefixen met actor. het gaat rap onoverzichtelij kworden anders vrees ik. zefde met die IType, iets te generiek